_posts/2014-07-31-dev-meetup-july-26-2014.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Monthly Dev Meetup July 2014"
+date: 2014-07-31 19:37
+comments: true
+author: "Saroj Maharjan"
+categories: [activities]
+author-twitter-name: "zoras"
+author-github-name: "zoras"
+---
+
+The tech scene in Kathmandu is ever growing but it is rather lacking a sense of community. Dev Meetup aims to fill this void with a regular meetup for developers where like minded individuals can gather, get to know each other and share their ideas/views.
+
+The meetup will be carried out monthly (last Saturday of the month) to cover web development topics like 'ruby', 'rails', 'javascript', 'devops', 'databases', etc.
+
+The first successful [Dev Meetup](https://www.facebook.com/events/768447783198796/){:.rnw-link} was held on July 26, 2014 at CloudFactory, the next Dev Meetup is scheduled for August 30th, 2014 at the same venue.
+
+Sign up at https://plus.google.com/events/cr78vpsi1466vv0ldj990fdgdr8 to join us. Kudos to CloudFactory for sponsoring both the events.
+
+Follow us [@ruby_nepal](http://twitter.com/ruby_nepal){:.rnw-link} for latest updates and news.
+
+If you are not sure what to expect or missed the 1st Dev Meetup, check out the pics and videos from the first one.
+
+** Pictures **
+
+![](https://pbs.twimg.com/media/BtdOPL3CMAAlPlN.jpg)
```

Comment:
okay @zoras  dai :)

Reply:
Hello @zoras  dai, I tried jekyll_figure plugin in such a way that ```https://pbs.twimg.com/media"``` added on dir section on _config.yml file. So that it works fine. But we have another source for images too i.e. ```http://photos4.meetupstatic.com/photos/event```.

I just look about we can use directly image link or not using this plugin, Implementation is done like this way in the plugin.

```
if defined? context.registers[:site].config["figures"]["dir"] 
        dir = context.registers[:site].config["figures"]["dir"]
      else
        dir = ""
end

**For the Image Source**
``` img_src = "#{dir}/#{filename}.#{extensions.first}"``` 
``` 
What do you suggest about it ? 
I get one alternative solution here ```https://thornelabs.net/2014/11/30/centering-images-with-jekyll-and-markdown.html```

Reply:
Update: 
<img width="840" alt="screen shot 2018-04-21 at 3 50 31 pm" src="https://user-images.githubusercontent.com/8276878/39082981-f10b8956-457b-11e8-9f05-85fe712b6599.png">